ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -651,21 +773,50 @@ This request is made with the following query params:
    </td>
   </tr>
   <tr>
-   <td>pre-<code>authorized_code</code>
+   <td><code>pre-authorized_code</code>
    </td>
    <td>Code representing the Credential Issuer's authorisation for the Wallet to obtain Credentials of a certain type. This code must be short-lived and single-use.
    </td>
   </tr>
   <tr>
-   <td><code>user_pin</code>
+   <td><code>tx_code</code>
```

Comment:
This newly introduced `tx_code` (replacing the `user_pin`) offers three helpful (but OPTIONAL) parameters such as 

- `input_mode `: OPTIONAL. String specifying the input character set. Possible values are numeric (only digits) and text (any characters). The default is numeric.

- `length`: OPTIONAL. Integer specifying the length of the Transaction Code. This helps the Wallet to render the input screen and improve the user experience.

- `description`: OPTIONAL. String containing guidance for the Holder of the Wallet on how to obtain the Transaction Code, e.g., describing over which communication channel it is delivered. The Wallet is RECOMMENDED to display this description next to the Transaction Code input screen to improve the user experience. 

Maybe we should consider using those parameters or at least the description one so that the wallet can present these information.

Reply:
Added direct reference to the standard instead of copy pasting.